### PR TITLE
[ADD] sales: creating quotations for 16

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/odoo/project/3835/tasks/4077995?cids=3

The focus of this PR is to ADD a piece of documentation to the Sales scope that focuses on the basic elements surrounding quotations and how to create them in Odoo Sales.

This PR uses the foundation of a similar PR made for 17.0, but makes the necessary version-specific adjustments to ensure the flow and screenshots are accurate for 16.0

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey @hojo-odoo -- this one is ready for a Peer Review, whenever you get a chance. Thanks!

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+In Odoo **Sales**, quotations can be created and sent to customers. Once a quotation has been
+confirmed, it officially turns into a *sales order*, which can then be invoiced and paid for.
+
+.. _sales/quotation-settings:
+
+Quotation settings
+==================
+
+To access these setting options, navigate to :menuselection:`Sales app --> Configuration -->
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:13]:
```suggestion
To access these settings options, navigate to :menuselection:`Sales app --> Configuration -->
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+
+- :guilabel:`Online Signature`: Request an online signature to confirm orders.
+- :guilabel:`Quotation Templates`: Enable this option to create quotation templates featuring
+  standard product offers, which are then selectable on quotation forms. When this checkbox is
+  ticked, an additional field, :guilabel:`Default Template`, appears, along with a link to the
+  :guilabel:`Quotation Templates` page.
+- :guilabel:`Default Quotation Validity`: Determine a set amount (in :guilabel:`days`) that
+  quotations can remain valid for.
+- :guilabel:`Lock Confirmed Sales`: Ensure no further edits can be made to confirmed orders.
+- :guilabel:`Online Payment`: Request an online prepayment from customers to confirm orders. Request
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:28]:
```suggestion
- :guilabel:`Online Payment`: Request an online prepayment from customers to confirm orders, which can be
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+  customers.
+- :guilabel:`Pro-Forma Invoice`: Send pro-forma invoices to customers.
+
+To activate any of these settings, tick the checkbox beside the desired option(s). Then, click
+:guilabel:`Save`.
+
+Quotations dashboard
+====================
+
+The *Quotations* dashboard is the page that appears when the :menuselection:`Sales app` is opened.
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:44]: I might be good to say how you can access the dashboard from elsewhere in the app.

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+   specific quotation template form.
+
+If the quotation is for a recurring product or subscription, select the desired
+:guilabel:`Recurrence` option from that specific drop-down menu.
+
+If a :guilabel:`Recurrence` option is selected, an :guilabel:`until` field appears beside it. In
+this field, select the desired date the :guilabel:`Recurrence` should end, via a calendar popover
+window.
+
+Also, if a :guilabel:`Recurrence` option is selected, a new form, :guilabel:`Freeze Plan` appears on
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:127]:
```suggestion
Also, if a :guilabel:`Recurrence` option is selected, a new field, :guilabel:`Freeze Plan`, appears on
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+Next, select a specific :guilabel:`Pricelist` to be applied to this quotation, if desired.
+
+Lastly, select any specific :guilabel:`Payment Terms` to be used for this quotation.
+
+Order Lines tab
+---------------
+
+The first tab on the quotation form is the :guilabel:`Order Lines` tab.
+
+In this tab, select products, and quantities of those products, to add to the quotation.
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:140]:
```suggestion
In this tab, select products, and the quantities of those products, to add to the quotation.
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+
+Click :guilabel:`Add a product`, select the desired item from the :guilabel:`Product` drop-down
+field, and proceed to adjust the quantity of that selected product, if necessary.
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:143]:
```suggestion
field, and proceed to adjust the :guilabel:`Quantity` of that selected product, if necessary.
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 7]
```diff
@@ -0,0 +1,326 @@
...
+
+.. note::
+   Some fields **only** appear if specific settings and options have been configured.
+
+.. _sales/sales-section:
+
+Sales section
+~~~~~~~~~~~~~
+
+In the :guilabel:`Sales` section of the :guilabel:`Other Info` tab, there are sales specific fields
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:189]:
```suggestion
In the :guilabel:`Sales` section of the :guilabel:`Other Info` tab, there are sales-specific fields
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 8]
```diff
@@ -0,0 +1,326 @@
...
+   :align: center
+   :alt: The Sales section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Salesperson`: Assign a salesperson from the drop-down menu to be associated with this
+  quotation. The user who originally created the quotation is selected in this field, by default.
+- :guilabel:`Sales Team`: Assign a specific sales team to this quotation. If the selected
+  :guilabel:`Salesperson` is a member of a sales team, that team is auto-populated in the field.
+- :guilabel:`Company`: Select a company from the drop-down menu this quotation should be associated
+  with. This field only appears when working in a multi-company environment.
+- :guilabel:`Online confiramtion: This field contains two checkboxes: :guilabel:`Signature` and
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:202]:
```suggestion
- :guilabel:`Online confirmation`: This field contains two checkboxes: :guilabel:`Signature` and
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 9]
```diff
@@ -0,0 +1,326 @@
...
+
+In the :guilabel:`Delivery` section of the :guilabel:`Other Info` tab, there are delivery-specific
+fields that can be configured.
+
+.. image:: create_quotations/other-info-delivery.png
+   :align: center
+   :alt: The Delivery section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Shipping Policy`: Select a desired shipping policy from the drop-down menu. If all
+  products are delivered at once, the delivery order is scheduled, based on the greatest product
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:228]:
```suggestion
  products are delivered at once, the delivery order is scheduled based on the greatest product
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 10]
```diff
@@ -0,0 +1,326 @@
...
+- :guilabel:`Delivery Date`: Click into the empty field to reveal a calendar popover, from which a
+  customer delivery date can be selected. If no custom date is required, refer to the
+  :guilabel:`Expected` date listed to the right of that field.
+
+.. _sales/invoice-payments-section:
+
+Invoicing and Payments section
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Invoicing` section of the :guilabel:`Other Info` tab, there are invoicing specific
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:240]:
```suggestion
In the :guilabel:`Invoicing` section of the :guilabel:`Other Info` tab, there are invoicing-specific
```

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 11]
```diff
@@ -0,0 +1,326 @@
...
+---------
+
+In the :guilabel:`Notes` tab of the quotation form, enter any specific internal notes about the
+quotation and/or customer, if desired.
+
+Sending and confirming quotations
+=================================
+
+Once all the necessary fields and tabs have been configured, it is time to send the quotation to the
+customer for confirmation. Upon confirmation, the quotation turns into an official sales order.
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:283]: Should "sales order" be abbreviated?

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 12]
```diff
@@ -0,0 +1,326 @@
...
+There is also a :guilabel:`Customer Preview` smart button present, in the upper-right corner of the
+form. When clicked, Odoo reveals a preview of the quotation the customer sees when they log into
+their customer portal. Click the :icon:`fa-arrow-right` :guilabel:`Back to edit mode` link at the
+top of the preview page, in the blue banner, to return to the quotation form.
+
+.. note::
+   If the *Lock Confirmed Sales* setting is enabled, the sales order becomes :guilabel:`Locked`, and
+   is indicated as such on the sales order form.
+
+At this point, the quotation has been confirmed, turned into a sales order, and is now ready to be
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:315]: Abbreviation

File: content/applications/sales/sales/send_quotations/create_quotations.rst [hunk 13]
```diff
@@ -0,0 +1,326 @@
...
+invoiced and paid for.
+
+For more information about invoicing, refer to the :doc:`Invoice based on delivered or ordered
+quantities <../invoicing/invoicing_policy>`
```
Comment (hojo-odoo) [inline content/applications/sales/sales/send_quotations/create_quotations.rst:319]:
```suggestion
quantities <../invoicing/invoicing_policy>` doc.
```

**Review by hojo-odoo — APPROVED** [review]
Hey @ksc-odoo,

Great job on this PR. It's ready for the next round of review, once you take a look at my comments.

Thanks,
John

**Comment by StraubCreative** [comment]
@samueljlieber no FWP on this one, 16 versions only plz 🙏

**Comment by samueljlieber** [comment]
@robodoo fw=no

File: content/applications/sales/sales/send_quotations/create_quotations.rst (hunk 3 above)
Comment (ksc-odoo) [reply content/applications/sales/sales/send_quotations/create_quotations.rst:44]: I had that in the v17 version of this doc, but was instructed to remove it during the review process, so I chose to keep that element out of this v16 version. But, if I get a similar note down the line, I'll be sure to add it in. No problem.

**Comment by ksc-odoo** [comment]
Thank you for the awesome feedback, @hojo-odoo -- I made all those suggested edits, and believe this one is now ready for a Final Review, @StraubCreative -- whenever you get a chance. Thanks! 👍

**Review by samueljlieber — APPROVED** [review]
Looks good to me @ksc-odoo! Nice job :)
.....
@robodoo r+

**Pull request closed without merging**